index.html

```diff
@@ -73,7 +73,7 @@ <h1>My Recent Works</h1>
   <!-- Main Card -->
   <div class="main-card">
     <img src="images/main-card-web-image.svg" alt="Exercise Poster" />
-    <div class="main-card-div">
+    <div class="main-card-div Card1">
```

Comment:
**_you did a great effort implementing the changes and i understand that you running out of time so i will mark this as OPTIONAL and i will approve the PR because you deserve it and you tried your best, but try to fix this isuue to make your website looks awesome_** 👏 👏 👏 👏 👏 

- [ ]  [OPTIONAL] great job putting the button back,  but when i hover over one card all of the cards in the same column get affected and resized as well , kindly try to fix it

![biggg](https://user-images.githubusercontent.com/89282221/187049176-aac6fcc3-77a7-464f-9d97-4c7b6a9c8f69.gif)

Reply:
Thank you 🤝 @ichala for your Comments. I shall surely Implement them.